Makefile

```
@@ -31,22 +31,21 @@ secret-scan:
 	trufflehog --max_depth 1 --exclude_paths trufflehog-ignore.txt .
 
 build-docs:
-	pip install -e ".[docs]"
+	uv pip install --system "kedro[docs] @ ."
```

Any reason to not do

```suggestion
	uv pip install --system -e ".[docs]"
```

?

---

Oh missed that one!